models/tokemak/ethereum/tokemak_ethereum_view_tokemak_lookup_reactors.sql

```diff
@@ -0,0 +1,68 @@
+{{ config (tags=['dunesql'],
+    schema='tokemak_ethereum',
+    alias = alias('view_tokemak_lookup_reactors'),
+    post_hook = '{{ expose_spells(\'["ethereum"]\',
+        "project", 
+            "Tokemak",
+             \'["addmorebass"]\') }}'
+) }}
+
+WITH tokemak_ethereum_view_tokemak_lookup_reactors
+(
+	    reactor_address, underlyer_address, reactor_name, is_deployable
+) AS (
+
+    SELECT '0xD3B5D9a561c293Fb42b446FE7e237DaA9BF9AA84' as reactor_address, '0xdBdb4d16EdA451D0503b854CF79D55697F90c8DF' as underlyer_address, 'ALCX Reactor', true
```

Comment:
We can use varbinary type for hexstring addresses.
```suggestion
    SELECT 0xD3B5D9a561c293Fb42b446FE7e237DaA9BF9AA84 as reactor_address, 0xdBdb4d16EdA451D0503b854CF79D55697F90c8DF as underlyer_address, 'ALCX Reactor', true
```